notebooks/official/ray_on_vertex_ai/get_started_with_pytorch_rov.ipynb

```diff
@@ -165,238 +172,188 @@
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "Create a storage bucket to store intermediate artifacts such as datasets."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "254614fa0c46"
+        "id": "MzGDU7TWdts_"
       },
       "outputs": [],
       "source": [
-        "# ! gcloud auth login"
+        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ef21552ccea8"
+        "id": "-EcIXiGsCePi"
       },
       "source": [
-        "**3. Colab, uncomment and run:**"
+        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "603adbbf0532"
+        "id": "NIq7R4HZCfIc"
       },
       "outputs": [],
       "source": [
-        "# from google.colab import auth\n",
-        "# auth.authenticate_user()"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "f6b2ccc891ed"
-      },
-      "source": [
-        "**4. Service account or other**\n",
-        "* See how to grant Cloud Storage permissions to your service account at https://cloud.google.com/storage/docs/gsutil/commands/iam#ch-examples."
+        "! gsutil mb -l {LOCATION} -p {PROJECT_ID} {BUCKET_URI}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "zgPO1eR3CYjk"
+        "id": "3d5191a94246"
       },
       "source": [
-        "### Create a Cloud Storage bucket\n",
-        "\n",
-        "Create a storage bucket to store intermediate artifacts such as datasets."
+        "### Initialize Vertex AI SDK for Python"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "MzGDU7TWdts_"
+        "id": "de483dc2a7ee"
       },
       "outputs": [],
       "source": [
-        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
+        "from google.cloud import aiplatform as vertex_ai\n",
+        "\n",
+        "vertex_ai.init(project=PROJECT_ID, location=LOCATION, staging_bucket=BUCKET_URI)"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "-EcIXiGsCePi"
+        "id": "454cb4f7a9e9"
       },
       "source": [
-        "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
+        "#### Timestamp\n",
+        "\n",
+        "If you are in a live tutorial session, you might be using a shared test account or project. To avoid name collisions between users on resources created, you create a timestamp for each instance session, and append the timestamp onto the name of resources you create in this tutorial."
```

Comment:
Remove "you" from "you create a timestamp:
- "To avoid name collisions between users on resources created, create a timestamp for each instance session, and append the timestamp onto the name of resources you create in this tutorial."